modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/NativePlatform.java

```diff
@@ -34,6 +37,23 @@
     private NativeScreen screen;
     protected AcceleratedScreen accScreen;
 
+
+    protected static final boolean useCursor =
+        AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> {
+            final String str =
+                System.getProperty("monocle.cursor.enabled", "true");
+            return "true".equalsIgnoreCase(str);
+        });
+
+    protected static final boolean debugCursor =
+        AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> {
+            final String str =
+                System.getProperty("monocle.debugcursor", "");
```

Comment:
Just a nit, but why `monocle.debugcursor` rather than `monocle.cursor.debug` (my preference), or at least `monocle.debugCursor`? Below is the full list, for comparison, including the two added by this pull request.

```
monocle.cursor.enabled
monocle.debugcursor
monocle.epd.bitsPerPixel
monocle.epd.enableInversion
monocle.epd.forceMonochrome
monocle.epd.noWait
monocle.epd.rotate
monocle.epd.useDitheringY1
monocle.epd.useDitheringY4
monocle.epd.waveformMode
monocle.epd.y8inverted
monocle.input.<product>.flipXY
monocle.input.<product>.maxX
monocle.input.<product>.maxY
monocle.input.<product>.minX
monocle.input.<product>.minY
monocle.input.<product>.touchFilters
monocle.input.touchFilters
monocle.input.touchRadius
monocle.input.traceEvents
monocle.input.traceEvents.verbose
monocle.maliSignedStruct
monocle.platform
monocle.platform.traceConfig
monocle.screen.fb
monocle.stackSize
```

I'm nervous about our hidden API of system properties, and I'm as guilty as anyone with the nine properties I added for Monocle EPD. I think it might be okay as long as the code gets the property values only during class initialization. That should restrict their use to startup scripts and keep them out of application code trying to modify them *on-the-fly* at run time.

Reply:
if i recall i originally started with the format you recommend as it made more sense, and when looking for other debug logging across the javafx stack I picked up on somewhat of a loose existing convention so changed to match it. I guess it can be whatever everyone agrees upon. :)

Reply:
On second thought, let's remove `monocle.debugcursor` and use a [PlatformLogger](https://github.com/openjdk/jfx/blob/master/modules/javafx.base/src/main/java/com/sun/javafx/logging/PlatformLogger.java). The JavaFX loggers are available from [Logging](https://github.com/openjdk/jfx/blob/master/modules/javafx.graphics/src/main/java/com/sun/javafx/util/Logging.java). In retrospect, that's how I managed to avoid any new *debug* properties for Monocle EPD even though it's packed with debugging and trace messages. For examples, see the variable `logger` in [EPDFrameBuffer](https://github.com/openjdk/jfx/blob/master/modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/EPDFrameBuffer.java), where `logger.fine` is called for messages printed once per run, while `logger.finer` is called for messages printed once per rendered frame.